auslib/blobs/schemas/systemaddons.yml

```diff
@@ -0,0 +1,71 @@
+---
+  title: SystemAddons Blob
+  type: object
+  required:
+    - name
+    - schema_version
+    - hashFunction
```

Comment:
For this blob, hashFunction is not actually required, because of the uninstall option. If it's possible to make it required only when addons is present, let's do that. Otherwise just remove it from here.